airflow-core/src/airflow/api_fastapi/core_api/datamodels/dags.py

```diff
@@ -180,3 +180,16 @@ def latest_dag_version(self) -> DagVersionResponse | None:
         if latest_dag_version is None:
             return latest_dag_version
         return DagVersionResponse.model_validate(latest_dag_version)
+
+
+class ReserializeResponse(BaseModel):
+    """DAG Reserialize serializer for responses."""
+
+    message: str
+    processed_bundles: list[str]
```

Comment:
I think we should unify with other 'list' response type and "Collections". You can compare with other collection serializers.

Reply:
You can look for `CollectionResponse` to find them. `AssetAliasCollectionResponse`, `AssetResponse`, `DAGResponse`, etc....